.github/workflows/solvcon_install.yml

```diff
@@ -3,7 +3,23 @@ on:
   push:
   pull_request:
 jobs:
-  build:
+  unittest:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
Please explicit use `ubuntu-20.04` instead of the latest tag, which usually results into surprises when the upstream CI decides to upgrade.

Reply:
No problem. Will do!